### PR TITLE
Compatibility with Laravel 6.0

### PR DESCRIPTION
**Pull request by benjamincrozat**
A little of my time to help you Spatie guys. It seems like the tests are still green. Please let me know about the versions constraints; maybe I was a little bit too hardcore, because I'm assuming a new major version (which would be 4.0).

### CODE REVIEW

**Comment by sixlive** [comment]
Awesome, thanks! I'm going to get around to reviewing all the open stuff on this repo over the next day or so!

File: composer.json [hunk 1]
```diff
@@ -19,15 +19,15 @@
         "php": "^7.2",
         "graham-campbell/guzzle-factory": "^3.0",
         "guzzlehttp/guzzle": "^6.3",
-        "laravel/framework": "~5.8.0",
```
Comment (brendt) [inline composer.json]: Please support `~5.8.0|^6.0`

File: composer.json [hunk 2]
```diff
@@ -19,15 +19,15 @@
...
+        "laravel/framework": "^6.0",
         "laravel/slack-notification-channel": "^2.0",
         "spatie/ssl-certificate": "^1.5.0",
         "spatie/url": "^1.0.1"
     },
     "require-dev": {
         "mockery/mockery": "^1.0",
-        "orchestra/testbench": "~3.8.0",
-        "phpunit/phpunit": "^7.5|^8.0"
+        "orchestra/testbench": "^4.0",
```
Comment (brendt) [inline composer.json]: Support both `~3.8.0|^4.0`

File: composer.json [hunk 3]
```diff
@@ -19,15 +19,15 @@
...
-        "laravel/framework": "~5.8.0",
+        "laravel/framework": "^6.0",
         "laravel/slack-notification-channel": "^2.0",
         "spatie/ssl-certificate": "^1.5.0",
         "spatie/url": "^1.0.1"
     },
     "require-dev": {
         "mockery/mockery": "^1.0",
-        "orchestra/testbench": "~3.8.0",
-        "phpunit/phpunit": "^7.5|^8.0"
```
Comment (brendt) [inline composer.json]: Don't change the phpunit version in this PR

File: .travis.yml [hunk 4]
```diff
@@ -11,13 +11,13 @@ env:
 matrix:
   include:
     - php: 7.2
-      env: LARAVEL='5.8.*' TESTBENCH='3.8.*' COMPOSER_FLAGS='--prefer-lowest'
```
Comment (brendt) [inline .travis.yml]: Support both 5.8 and 6.0

**Comment by brendt** [comment]
Let's support both for now.

**Comment by benjamincrozat** [comment]
Alright, done!

**Comment by brendt** [comment]
The Travis matrix config is still wrong. Sorry for the late reply, I wasn't at the office the past few days.

**Comment by benjamincrozat** [comment]
Sorry, totally forgot about it. Here's the missing commit.

**Comment by brendt** [comment]
Tagged as 3.6.0: https://github.com/spatie/laravel-uptime-monitor/releases/tag/3.6.0

**Pull request merged**